### PR TITLE
qu-auto/SORQA-367 Automatize "Immunizations III: Immunization list for cases, contacts and event participants [5]"

### PR DESCRIPTION
**Pull request by hms-sgent**
Fixes #

### CODE REVIEW

**Comment by hms-sgent** [comment]
https://jenkins.sormas.netzlink.com/job/sormas-RunE2eTests-branch/1445/
https://jenkins.sormas.netzlink.com/job/sormas-RunE2eTests-branch/1446/
https://jenkins.sormas.netzlink.com/job/sormas-RunE2eTests-branch/1448/

**Comment by hms-sgent** [comment]
image

**Review by rdutu-vg — CHANGES_REQUESTED** [review]

File: sormas-e2e-tests/src/test/resources/features/sanity/web/Immunization.feature [hunk 1]
```diff
@@ -136,6 +136,69 @@ Feature: Immunization end to end tests
     And I check if External ID input on immunization edit page is disabled
     And I check if Additional details text area on immunization edit page is disabled
 
+  @issue=SORDEV-8059 @env_main
+  Scenario: Immunization III: List for cases, contacts and event participants
```
[the remark this answers is not in the document]
Comment (hms-sgent) [reply sormas-e2e-tests/src/test/resources/features/sanity/web/Immunization.feature]: done

**Comment by hms-sgent** [comment]
https://jenkins.sormas.netzlink.com/job/sormas-RunE2eTests-branch/1473/

**Review by rdutu-vg — APPROVED** [review]

**Pull request merged**